baseline/dev-tools/local-server/webpack.config.js

```diff
@@ -0,0 +1,53 @@
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const path = require("path");
+
+const client = path.join(__dirname, "../../client");
+
+module.exports = {
+    entry: {
+        flanker: {
+            import: path.join(client, "flanker/flanker.js"),
+            filename: "flanker/flanker.bundle.js",
+        },
+    },
+    plugins: [
+        new HtmlWebpackPlugin({
+            title: "Flanker Task",
+            filename: "flanker/index.html",
+            chunks: ["flanker"],
+        }),
+        new HtmlWebpackPlugin({
+            filename: "index.html",
+            template: path.join(__dirname, "public/index.html"),
+            chunks: [],
+        }),
+    ],
+    module: {
+        rules: [
+            {
+                test: /\.html$/i,
+                loader: "html-loader",
+            },
+            {
+                test: /\.css$/i,
+                use: ["style-loader", "css-loader"],
+            },
+            {
+                test: /\.(png|svg|jpg|jpeg|gif)$/i,
+                type: 'asset/resource',
+            },
+        ],
+    },
```

Comment:
These rules determine how to handle imports.

https://webpack.js.org/configuration/module/